javascript/chronometer.js

```diff
@@ -1,26 +1,32 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
   startClick(callback) {
-    // ... your code goes here
-  }
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+    }, 1000);
+    }
+  
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime/60);
   }
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 60);
   }
   twoDigitsNumber() {
-    // ... your code goes here
+    if (this.currentTime < 10) {
+      return ("0" + this.currentTime);
+    }
   }
```

Comment:
It's better practice if you pass "this.currentTIme" as an argument to be passed to the function `twoDigitsNumber(timeValue){...}`

You should also have an `else{ return timeValue}`. It can all be rewritten using the ternary operator 
```
twoDigitsNumber(timeValue) {
return timeValue < 10 ? "0" + timeValue : timeValue;
}